src/Helpers/PageHelpers/PageHelper.cs

```diff
@@ -137,14 +137,17 @@ public void SaveAnswers(Dictionary<string, dynamic> viewModel, string guid, stri
             _distributedCache.SetStringAsync(guid, JsonConvert.SerializeObject(convertedAnswers));
         }
 
-        public void SaveCaseReference(string guid, string caseReference)
+        public void SaveCaseReference(string guid, string caseReference, bool isGenerated=false, string generatedRefereceMappingId="GeneratedReference")
```

Comment:
We need spaces around the equals for these default values

Reply:
Done